spaces/S000110/properties/P000093.md

```diff
@@ -0,0 +1,7 @@
+---
+space: S000110
+property: P000093
+value: true
+---
+
+Every $n \in \omega$ has $\{\omega\}$ as a countable neighborhood and every $F \in M$ has $\omega \cup \{F\}$ as a countable neighborhood.
```

Comment:
Did you mean to write: Every $n \in \omega$ has $\\{n\\}$ as a countable neighborhood?

Reply:
@GeoffreySangston Ah, I meant the first one, but I was thinking of both at the same time, so I just accidentally combined the two lol. I'll make an edit.